Added an readme of kebechet components (#591)
thoth-station/thoth-application: kebechet/README.md

* **prod**: ocp.prod cluster, `aicoe-prod-bots` namespace

Default configuration for all CronJobs is: suspended, prod deployment patches that and activates the CronJobs.

We run Kebechet in two different modes -

> * The default mode i.e. via an webhook trigger from the Github app

Reviewer: ```suggestion
* The default mode i.e. via a webhook trigger from the Github app
```

>   The template for that is [here](https://github.com/thoth-station/thoth-application/blob/master/kebechet/base/openshift-templates/kebechet.yaml). Here kebechet acts on an webhook and operates on the particular repository. In this run we just need the webhook payload from the Github app.

Reviewer: ```suggestion
  The template for that is [here](https://github.com/thoth-station/thoth-application/blob/master/kebechet/base/openshift-templates/kebechet.yaml). Here kebechet acts on a webhook and operates on the particular repository. In this run, we just need the webhook payload from the Github app.
```

* The other way is we initiate Kebechet run on a user repository when we recieve new information related to the particular user. We initiate Kebechet the particular repository using the [Kebechet-run-url](https://github.com/thoth-station/thoth-application/blob/master/kebechet/base/openshift-templates/kebechet-run-url.yaml) template. In this run we just need the repository name and the service type which is Github in this case.

> Kebechet now has an Administrator component that finds out all the repositories that have been impacted by a particular event and schedules an Kebechet run on them. This component used the base image of workflow helper with the subcommand `kebchet_administrator`. Template link - [here](https://github.com/thoth-station/thoth-application/blob/master/kebechet/base/openshift-templates/kebechet-administrator.yaml)

Reviewer: ```suggestion
Kebechet now has an Administrator component that finds out all the repositories that have been impacted by a particular event and schedule a Kebechet run for each of them. This component uses the base image of the workflow helper with the subcommand `kebchet_administrator`. Template link - [here](https://github.com/thoth-station/thoth-application/blob/master/kebechet/base/openshift-templates/kebechet-administrator.yaml)
```
